hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -120,39 +127,75 @@ public void start() {
     }
 
     private void startNio() {
+        TPCSocketConfig clientSocketConfig = getClientSocketConfig();
+
+        if (clientSocketConfig == null) {
+            // advanced network is enabled yet there is no configured server socket for clients
```

Comment:
is ... enabled? not?

Reply:
https://github.com/hazelcast/hazelcast/pull/23254/commits/d4a32e5485257ba18595fb0858b03756aa8321ed